apps/messaging-api/routes/users/index.ts

```diff
@@ -199,4 +191,7 @@ export default async function users(app: FastifyInstance) {
       return { data: user };
     },
   );
+
+  const parseActiveOnlyParam = (activeOnly: boolean | undefined): boolean =>
+    typeof activeOnly === "undefined" ? true : activeOnly;
```

Comment:
This can be simplified to:

```suggestion
   activeOnly ?? true;
```